specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2024-07-10/privateLinkScopes.json

```diff
@@ -0,0 +1,1714 @@
+      "enum": [
+        "Inbound",
+        "Outbound"
+      ],
+      "x-ms-enum": {
+        "name": "AccessRuleDirection",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Inbound",
+            "description": "Traffic originates outside of network."
+          },
+          {
+            "value": "Outbound",
+            "description": "Traffic originates inside the network"
+          }
+        ]
+      }
+    },
+    "AccessRule": {
+      "type": "object",
+      "description": "Access rule.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the access rule.",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Access rule properties",
+          "readOnly": true,
+          "$ref": "#/definitions/AccessRuleProperties"
+        }
+      }
+    },
+    "AccessRuleProperties": {
+      "type": "object",
+      "description": "Properties of an access rule.",
+      "properties": {
+        "direction": {
+          "description": "Direction of the access rule.",
+          "readOnly": true,
+          "$ref": "#/definitions/AccessRuleDirection"
+        },
+        "addressPrefixes": {
+          "type": "array",
+          "description": "Address prefixes that are allowed access.",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "NetworkSecurityPerimeter": {
+      "type": "object",
+      "description": "Properties that define a Network Security Perimeter resource.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "perimeterGuid": {
+          "type": "string",
+          "description": "Guid of the Network Security Perimeter",
+          "readOnly": true
+        },
+        "location": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Regional location of the perimeter"
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationListResult": {
+      "type": "object",
+      "description": "A list of network security perimeter configurations.",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkSecurityPerimeterConfiguration"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationReconcileResult": {
+      "type": "object",
+      "description": "Result of network security perimeter configurations.",
+      "properties": {
+        "location": {
```

Comment:
>location [](http://example.com/codeflow?start=9&length=8)

sound like you can add the uri format, also add the  x-ms-mutability attribute

Reply:
added to the definition